tests/src/common/WskCli.java

```diff
@@ -329,12 +329,19 @@ public String updateAction(String name, boolean shared) throws IOException {
         return createAction(SUCCESS_EXIT, name, null, null, null, false, true, false, shared, 0);
     }
 
+    public String updateAction(String name, String file, String kind) throws IOException {
+        return createAction(SUCCESS_EXIT, name, file, null, null, false, true, false, false, 0, kind);
+    }
+
```

Comment:
Add the test to the scala basic suite suite instead and then you don't need these changes.
See https://github.com/openwhisk/openwhisk/issues/786 - we should avoid adding Java tests unless necessary.